### PR TITLE
Add deploy to ECR workflow.

### PR DESCRIPTION
**Pull request by verdammelt**
Blocked by exercism/terraform#14

### CODE REVIEW

**Comment by verdammelt** [comment]
@ErikSchierboom can I get your review of this? Thanks.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by verdammelt** [comment]
@iHiD / @TheLostLambda  can you review and/or merge? I thought I'd be able to merge after @ErikSchierboom gave it his blessing but that didn't seem to be enough.

**Comment by ErikSchierboom** [comment]
@verdammelt I don't have write permissions, hence my approval being not enough.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @exercism/maintainers-admin
+* @verdammelt @TheLostLambda @exercism/maintainers-admin
```
Comment (iHiD) [inline .github/CODEOWNERS:1]: I'd rather it was this. This means we don't get notified about stuff.

```suggestion
* @verdammelt @TheLostLambda 
.github/CODEOWNERS @exercism/maintainers-admin
```

**Comment by verdammelt** [comment]
@iHiD thanks for catching that - fixed to your suggestion.

**Review by TheLostLambda — APPROVED** [review]
This looks good to me! (Same as what I did for the test-runner a little while ago)

**Pull request merged**